repairnator/repairnator-pipeline/src/test/java/fr/inria/spirals/repairnator/process/files/FileHelperTest.java

```diff
@@ -0,0 +1,62 @@
+package fr.inria.spirals.repairnator.process.files;
+
+import static org.junit.Assert.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.After;
+import org.junit.Test;
+
+import fr.inria.spirals.repairnator.process.files.FileHelper;
+import fr.inria.spirals.repairnator.process.inspectors.JobStatus;
+import fr.inria.spirals.repairnator.process.inspectors.ProjectInspector;
+import fr.inria.spirals.repairnator.process.step.CloneRepository;
+import fr.inria.spirals.repairnator.process.utils4tests.ProjectInspectorMocker;
+
+
+public class FileHelperTest {
+	
+	private File tmpDir;
+	
+	@After
+	public void tearDown() throws IOException {
+	    FileHelper.deleteFile(tmpDir);
+	}
+	
+	@Test
+	public void testRemoveNotificationFromTravisYML() throws IOException{
+	    File resourceFile = new File("./src/test/resources/travis-file/.travis.yml");
+	    File tmpDir = Files.createTempDirectory("test_removeNotificationFromTravisYML").toFile();
```

Comment:
Since you have a field `tmpDir`, you don't need it as a local variable.

Reply:
Fixed.